docs/docs/graphql/permissions.mdx

```diff
@@ -8,35 +8,19 @@ The GraphQL API is protected by a role-based permission system.
 
 For each **role**, you create **rules** for the **select**, **insert**, **update**, and **delete** operations.
 
-**Example:** Let's say you have a `posts` table, and you want users to only access their own posts. This is how you would do it:
-
-```sql title="Posts Table"
-CREATE TABLE "public"."posts" (
-  "id" serial NOT NULL PRIMARY KEY,
-  "title" text NOT NULL,
-  "user_id" uuid NOT NULL,
-);
-```
+**Example:** Let's say you have a `posts` table with `id` `title` and `user_id` columns, and you want users to only access their own posts. This is how you would do it:
 
-```json title="Hasura Permission Rule"
-{
-  "user_id": {
-    "_eq": "X-Hasura-User-Id"
-  }
-}
-```
+![Permission Variables](/img/graphql/permissions/permission-example.png)
 
-The rule above make it so users can only select posts where the value of `user_id` is equal (`_eq`) to their user ID (`x-hasura-user-id`).
+The permission above make sure users can only select their own posts, because the value of `user_id` must be equal (`_eq`) to the authenticated user's ID (`x-hasura-user-id`).
```

Comment:
```suggestion
The permission above makes sure users can only select their own posts, because the value of `user_id` must be equal (`_eq`) to the authenticated user's ID (`x-hasura-user-id`).
```